src/style/panel/panel.less

```diff
@@ -81,7 +81,7 @@
     &--move-in--right {
       & .panel__content,
       & .panel__footer {
-        transform: translateX(-var(--panel-width));
+        transform: translateX(calc(var(--panel-width) * -1));
```

Comment:
* -1 is a hack? what does it do? :D

Reply:
Why not create another variable like:
`--nagative-panel-width: -304px;`
Just a suggestion.